### PR TITLE
feat: Adding captions, drag and drop upload, url and Mux ID inputs, a…

### PR DESCRIPTION
**Pull request by blairliikala**
…udio normalization, player code

chore: upgraded libraries including React, Contentful, and testing.

### CODE REVIEW

File: apps/mux/src/locations/config.tsx [hunk 1]
```diff
@@ -209,7 +227,7 @@ class Config extends React.Component<ConfigProps, IState> {
       <React.Fragment>
         <div className="config-background" />
         <div className="config-body">
-          <Typography>
+          <React.Fragment>
```
Comment (andipaetzold) [inline apps/mux/src/locations/config.tsx:230]: I think you can remove the Fragments in this file?

File: apps/mux/src/index.tsx [hunk 2]
```diff
@@ -1,61 +1,48 @@
...
+  Heading,
+  Box,
+  Flex,
+  Switch,
+} from '@contentful/f36-components';
+import { Form, FormControl, Checkbox, TextInput } from '@contentful/f36-forms';
+
+import MuxPlayer from '@mux/mux-player-react/dist/index.cjs';
+import MuxUploader from '@mux/mux-uploader-react/dist/index.cjs';
+import { MuxUploaderDrop } from '@mux/mux-uploader-react/dist/index.cjs';
```
Comment (andipaetzold) [inline apps/mux/src/index.tsx:20]: Doesn't this work?
```suggestion
import MuxPlayer from '@mux/mux-player-react';
import MuxUploader from '@mux/mux-uploader-react';
import { MuxUploaderDrop } from '@mux/mux-uploader-react';
```

**Review by andipaetzold — APPROVED** [review]
You changed permissions of some files (100644 → 100755). Was that intentional?

I didn't do an in-depth review due to the PR size, ideally someone from Mux should have a look.

File: apps/mux/src/locations/config.tsx (hunk 1 above)
Comment (blairliikala) [reply apps/mux/src/locations/config.tsx]: Yep!  Removed.

File: apps/mux/src/index.tsx (hunk 2 above)
Comment (blairliikala) [reply apps/mux/src/index.tsx:20]: Unfortunately for the moment we have to use the cjs import. Once we can upgrade to React Scripts v5 then we can switch back.

**Comment by andipaetzold** [comment]
@blairliikala Just merge master into this branch and I can merge

**Comment by blairliikala** [comment]
> @blairliikala Just merge master into this branch and I can merge

Just checking, you did merge this into master already, or did you need me to merge?

**Comment by andipaetzold** [comment]
@blairliikala Just merged myself. Didn't know I had permissions to do that.

**Pull request merged**